packages/web3-eth-accounts/src/account.ts

```diff
@@ -63,17 +214,11 @@ export const privateKeyToAccount = (
 export const create = (): {
 	address: HexString;
 	privateKey: string;
-	signTransaction: () => boolean; // From 1.x
-	sign: () => boolean;
+	signTransaction: signTransactionFunction; // From 1.x
+	sign: signFunction;
 	encrypt: () => boolean;
 } => {
```

Comment:
You don't need to specify the return object like this, TypeScript will automatically infer it

Reply:
ts will infer but good to mention for user's usage or visibility